gocmd/gocmd_test.go

```diff
@@ -19,3 +19,62 @@ func TestModule(t *testing.T) {
 
 	assert.Equal(t, "github.com/mt-sre/go-ci", module)
 }
+
+// TestTidyConfig_Option tests the Option method of the
+// TidyConfig struct, which allows setting configuration options
+func TestTidyConfig_Option(t *testing.T) {
+	config := &TidyConfig{}
+
+	withGoVersion := func(version string) TidyOption {
+		return tidyOptionFunc(func(c *TidyConfig) {
+			c.GoVersion = version
+		})
+	}
+
+	withWorkingDir := func(dir string) TidyOption {
+		return tidyOptionFunc(func(c *TidyConfig) {
+			c.WorkingDir = dir
+		})
+	}
```

Comment:
It's probably better to just implement these as options [here](https://github.com/mt-sre/go-ci/blob/42a771b0ab5ed350136ba1db144cbe5351862c1d/gocmd/options.go). Then you can use them in tests or as a user of the library.